.github/workflows/cd.yml

```diff
@@ -168,122 +121,119 @@ jobs:
           OUT_DIR=release/completions/ cargo run --release --bin git-cliff-completions
           OUT_DIR=release/man/ cargo run --release --bin git-cliff-mangen
           for bin in 'git-cliff' 'git-cliff-completions' 'git-cliff-mangen'; do
-            if [ "${{ matrix.build.OS }}" = "windows-2022" ]; then
+            if [ "${{ contains(matrix.os, 'windows') }}" = "true" ]; then
```

Comment:
This can be further simplified and deduplicated by splitting the os and runner in the matrix above, either by adding a `runner` variable, or by adding an `is_windows` variable. I didn't worry about this as it's clear enough here.

There's possibly also a bash test that checks for `true` instead of string comparisonI but I haven't looked.